overview/models.py

```diff
@@ -140,6 +141,11 @@ def twitter_url(self):
         if self.twitter:
             return "https://twitter.com/{}".format(self.twitter)
 
+    @property
+    def total_contributions_less_fees(self):
+        contributions = PastContribution.objects.filter(candidate=self).exclude(note__contains='access fee').aggregate(Sum('amount'))
```

Comment:
Doesn't matter, but just in case you didn't know, the foreign keys define "related managers" so this could have been

```
self.pastcontribution_set.exclude(...)...
```

Reply:
This will return None if there are no contributions which is probably handled fine by floatformat / intcomma

Reply:
woops. I see now that you know about self.pastcontribution_set